quiz.css

```diff
@@ -0,0 +1,69 @@
+/*The entire card has a solid 1px border that is lightblue.
+The card title has a dotted 1px border that is lightgray.
+The title and product image are centered.
+Notice that the text for the description in the image is justified.
+The text for the product specification details and pricing details is bold.
+The availability element extends the full width of the card, with a dark grey background and yellow text.*/
+
+/*  base element selectorss  */
+
+* {
```

Comment:
**Pro tip:** The `*` selector in CSS is a very expensive (i.e. memory intensive) operation for the browser.  Be as specific as you can possibly be with your CSS selectors.  You may want to consider using `body` here instead.